src/components/work/ProfilePhotoUploader/ProfilePhotoUploader.container.ts

```diff
@@ -0,0 +1,64 @@
+import styled from 'styled-components';
+import { color, typography } from 'styled/utils';
+
+const ProfilePhotoUploaderContainer = styled.div`
+  .photo-uploader-container {
```

Comment:
i assume this is a copy and paste from old component?

Reply:
mhm

Reply:
it's ok it is a copy/pasta but if it's completely unchanged, can't we just use the existing component?

Reply:
ooo the old one has media queries where there's a display none when the screen is less than desktop. In this implementation in an effort to shy away from media queries, the photo uploader is always shown and I removed that display none property.

If the old component was used, there would be an empty `<Card />` component (it wraps ProfilePhotoUploader) on Tablet and Mobile.

Reply:
![image](https://user-images.githubusercontent.com/18321302/53529388-4b5c5a00-3aa1-11e9-90f5-3a965644ce16.png)

Reply:
ok, ideally we want to stick to the components front already provided for us to avoid re-writing CSS.
see:
![screen shot 2019-02-27 at 3 22 51 pm](https://user-images.githubusercontent.com/7976/53530070-94ada900-3aa3-11e9-8bec-12e33aa01b94.png)

The photo uploader can just be a button "Change Photo" vs drag and drop to save effort.

Reply:
Currently, the networking to upload to our backend and S3 is happening in the `<PhotoUploader />` component. Should I try to route an input field through that component or perhaps make a new gql endpoint just for photo uploading?

Reply:
can we change the styling without changing the functionality?

Reply:
i think we can do the same thing we're doing in the old header where it checks the db, firebase, and then returns the default photo.  that way, we can ensure we get an updated photo once the DB updates

Reply:
don't think so since the styles are coupled within the PhotoUploader Component, I can try to recreate new component: the Dropzone component that use bootstrap styles only?

Reply:
let's just hold off on it for now. this PR looks fine with the current profile upload design.

Reply:
ok sg